package.json

```diff
@@ -3,25 +3,34 @@
   "version": "0.1.0",
   "description": "js monitor",
   "main": "index.js",
+  "repository": "apache/skywalking-client-js",
+  "homepage": "skywalking.apache.org",
+  "license": "Apache 2.0",
+  "bugs": {
+    "url": "https://github.com/apache/skywalking/issues",
+    "email": "dev@skywalking.apache.org"
+  },
   "dependencies": {
+    "typescript": "^3.7.2"
```

Comment:
Shouldn't this go to the `devDependencies`? It's not runtime dependency

Reply:
done.